src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+public class Duke {
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Initializes the Duke chatbot.
+     * @param filePath Specifies the path to save the content.
+     */
+
+    public Duke(String filePath) {
+        ui = new Ui();
+        tasks = new TaskList();
+        storage = new Storage(filePath);
+    }
+
+    /**
+     * start the program
+     */
+    public void run() {
+        ui.greetings();
+        while (true) {
+            String input = ui.readCommand();
+            String command = input.split(" ")[0];
+            switch (command) {
```

Comment:
Perhaps, the commands may be better represented using enums to ensure that only commands included in the enum is considered as a valid command. Another idea would be to create classes for the specific types of commands.